test/mux.sh

```diff
@@ -367,9 +334,9 @@ cleanup() {
         os::log::debug "$( oc replace --force -f $saveds )"
     fi
     # delete indices created by this test
-    curl_es $es_svc /project.testproj.* -XDELETE
-    curl_es $es_svc /project.default.* -XDELETE
-    curl_es $es_svc /project..orphaned.* -XDELETE
+    curl_es $es_pod /project.testproj.* -XDELETE
```

Comment:
I think these should be es_svc instead of es_pod

Reply:
They need to be, or they should be `curl_es_pod`